core/io/gets_spec.rb

```diff
@@ -223,7 +223,9 @@
 describe "IO#gets" do
   before :each do
     @name = tmp("io_gets")
-    touch(@name) { |f| f.write "朝日" + "\xE3\x81" * 100 }
+    # create filename "朝日" + "\xE3\x81" * 100 to avoid utf-8 conflicts
+    filename = "朝日" + ([227,129].pack('C*') * 100).force_encoding('utf-8')
```

Comment:
This is not a filename, it's the contents to be written and read from the file. Could you name it `contents` or `data`?